_data/locales/de.yml

```diff
@@ -3,27 +3,27 @@ de:
   subtitle: Der fehlende Paketmanager für macOS
   pagecontent:
     question: Was macht Homebrew?
-    what: Homebrew installiert <a href="https://formulae.brew.sh/formula/" title="Liste der Homebrew-Pakete">die Sachen, die du brauchst</a> und die bei Apple nicht dabei sind.
-    how: Homebrew installiert Pakete in ihrem eigenen Verzeichnis und erstellt  Symlinks zu ihren Dateien in <code>/usr/local</code>.
+    what: Homebrew installiert <a href="https://formulae.brew.sh/formula/" title="Liste der Homebrew-Pakete">Zeug, das du brauchst</a>, die Apple aber nicht mitliefert.
+    how: Homebrew installiert Pakete in ihrem eigenen Verzeichnis und erstellt Symlinks zu ihren Dateien in <code>/usr/local</code>.
     prefix: Homebrew installiert keine Dateien außerhalb seines Pfades und du kannst den Ort einer Homebrew-Installation frei wählen.
     createpackages: Eigene Homebrew-Pakete zu erzeugen, ist kinderleicht.
-    hack: Unter der Haube werden Git und Ruby verwendet. Modifikationen sind also schnell rückgängig gemacht und Änderungen lassen sich leicht zusammenfügen.
+    hack: Unter der Haube werden Git und Ruby verwendet. Modifikationen sind also schnell rückgängig gemacht und Änderungen lassen sich leicht zusammenführen.
     formula: "Homebrew-Formeln sind einfache Ruby-Skripte:"
-    editor: wird in $EDITOR geöffnet!
+    editor: Wird im $EDITOR geöffnet!
     complement: Homebrew ergänzt macOS. Installiere deine Gems mit <code>gem</code> und ihre Abhängigkeiten mit <code>brew</code>.
-    caskinstall: '"To install, drag this icon..." no more. <code>brew cask</code> installs macOS apps, fonts and plugins and other non-open source software.'
-    caskcreate: Making a cask is as simple as creating a formula.
+    caskinstall: 'Nie wieder "Ziehen Sie zum Installieren von ... dieses Icon ...". <code>brew cask</code> installiert macOS-Apps, -Schriftarten und -Plugins, sowie andere nicht-Open-Source-Software.'
```

Comment:
I was browsing Google images for German versions of those instructions. I'm on a Mac set to English.

Reply:
I don't think there is a canonical version of these instructions, and most of the time these are only in English since you can't translate a DMG background image.